staging/src/k8s.io/legacy-cloud-providers/azure/azure_cache.go

```diff
@@ -122,3 +122,12 @@ func (t *timedCache) Delete(key string) error {
 		key: key,
 	})
 }
+
+// Set sets the data cache for the key.
+// It is only used for testing.
```

Comment:
if it's only for testing, I think you could put this function in the test file since they are both in `Azure` package.

Reply:
the function is part of cache. Putting in test file would make it non clear.